tests/tracing/test_utils.py

```diff
@@ -15,19 +15,19 @@ def test_util_formats_info_path_value_into_reversed_list():
 
 
 def test_introspection_check_returns_true_for_introspection_field():
-    path = Mock(key="__types")
+    path = Mock(key="__type")
```

Comment:
`__types` is not an introspection field, as per the specification. This came up given that I made the checking more specific.

Reply:
Nice catch!